library/ssl_misc.h

```diff
@@ -611,7 +611,8 @@ struct mbedtls_ssl_handshake_params
      * Handshake specific crypto variables
      */
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3)
-    int tls13_kex_modes; /*!< key exchange modes for TLS 1.3 */
+    uint8_t key_exchange_mode; /*!< Selected key exchange mode */
```

Comment:
```suggestion
    uint8_t key_exchange_mode; /*!< Selected negotiation key exchange mode */
```

Reply:
This makes it more confusing to me. I hesitated between "Negotiated" and "Selected" but opted for "Selected" as it is not really negotiated in the sense that the server does not send its key exchange mode choice to the client. It is deduced from the content of the ClientHello/ServerHello and the endpoint configuration. All together I prefer to keep it as it is.

Reply:
Got it.